src/main/java/mate/academy/util/ConnectionUtil.java

```diff
@@ -0,0 +1,26 @@
+package mate.academy.util;
+
+import java.sql.Connection;
+import java.sql. DriverManager;
+import java. sql. SQLException;
```

Comment:
There are unnecessary spaces in the import statements for `DriverManager` and `SQLException`. Consider removing these spaces to maintain consistent formatting.